2024-02-04 16:48:58,219 INFO    MainThread:26015 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-02-04 16:48:58,219 INFO    MainThread:26015 [wandb_setup.py:_flush():76] Configure stats pid to 26015
2024-02-04 16:48:58,219 INFO    MainThread:26015 [wandb_setup.py:_flush():76] Loading settings from /Users/lihongxuan/.config/wandb/settings
2024-02-04 16:48:58,219 INFO    MainThread:26015 [wandb_setup.py:_flush():76] Loading settings from /Users/lihongxuan/Desktop/AIPI/Courses/AIPI540/AIPI-540/wandb/settings
2024-02-04 16:48:58,219 INFO    MainThread:26015 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-04 16:48:58,219 INFO    MainThread:26015 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-04 16:48:58,219 INFO    MainThread:26015 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-04 16:48:58,219 INFO    MainThread:26015 [wandb_init.py:_log_setup():526] Logging user logs to /Users/lihongxuan/Desktop/AIPI/Courses/AIPI540/AIPI-540/wandb/run-20240204_164858-8gkiu4hf/logs/debug.log
2024-02-04 16:48:58,219 INFO    MainThread:26015 [wandb_init.py:_log_setup():527] Logging internal logs to /Users/lihongxuan/Desktop/AIPI/Courses/AIPI540/AIPI-540/wandb/run-20240204_164858-8gkiu4hf/logs/debug-internal.log
2024-02-04 16:48:58,219 INFO    MainThread:26015 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f87d93eb9a0>
2024-02-04 16:48:58,219 INFO    MainThread:26015 [wandb_init.py:init():566] calling init triggers
2024-02-04 16:48:58,220 INFO    MainThread:26015 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-02-04 16:48:58,220 INFO    MainThread:26015 [wandb_init.py:init():616] starting backend
2024-02-04 16:48:58,220 INFO    MainThread:26015 [wandb_init.py:init():620] setting up manager
2024-02-04 16:48:58,221 INFO    MainThread:26015 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-02-04 16:48:58,222 INFO    MainThread:26015 [wandb_init.py:init():628] backend started and connected
2024-02-04 16:48:58,239 INFO    MainThread:26015 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-02-04 16:48:58,239 INFO    MainThread:26015 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-04 16:48:58,239 INFO    MainThread:26015 [wandb_init.py:init():720] updated telemetry
2024-02-04 16:48:58,352 INFO    MainThread:26015 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-04 16:48:58,690 INFO    MainThread:26015 [wandb_run.py:_on_init():2254] communicating current version
2024-02-04 16:48:58,795 INFO    MainThread:26015 [wandb_run.py:_on_init():2263] got version response 
2024-02-04 16:48:58,795 INFO    MainThread:26015 [wandb_init.py:init():804] starting run threads in backend
2024-02-04 16:49:13,535 INFO    MainThread:26015 [wandb_run.py:_console_start():2233] atexit reg
2024-02-04 16:49:13,535 INFO    MainThread:26015 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-02-04 16:49:13,536 INFO    MainThread:26015 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-02-04 16:49:13,536 INFO    MainThread:26015 [wandb_run.py:_redirect():2178] Redirects installed.
2024-02-04 16:49:13,536 INFO    MainThread:26015 [wandb_init.py:init():847] run started, returning control to user process
2024-02-04 17:27:57,947 INFO    MainThread:26015 [wandb_run.py:_finish():1962] finishing run aipi549/aipi540/8gkiu4hf
2024-02-04 17:27:57,947 INFO    MainThread:26015 [jupyter.py:save_history():445] not saving jupyter history
2024-02-04 17:27:57,947 INFO    MainThread:26015 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-04 17:27:57,947 INFO    MainThread:26015 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-04 17:27:57,947 INFO    MainThread:26015 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-02-04 17:27:57,948 INFO    MainThread:26015 [wandb_run.py:_restore():2185] restore
2024-02-04 17:27:57,948 INFO    MainThread:26015 [wandb_run.py:_restore():2191] restore done
2024-02-04 17:28:01,594 WARNING MsgRouterThr:26015 [router.py:message_loop():77] message_loop has been closed
2024-02-04 17:28:01,980 INFO    MainThread:26015 [wandb_run.py:_restore():2185] restore
2024-02-04 17:28:01,980 INFO    MainThread:26015 [wandb_run.py:_restore():2191] restore done
2024-02-04 18:10:18,491 INFO    MainThread:26015 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-02-04 18:10:18,491 INFO    MainThread:26015 [wandb_setup.py:_flush():76] Configure stats pid to 26015
2024-02-04 18:10:18,491 INFO    MainThread:26015 [wandb_setup.py:_flush():76] Loading settings from /Users/lihongxuan/.config/wandb/settings
2024-02-04 18:10:18,492 INFO    MainThread:26015 [wandb_setup.py:_flush():76] Loading settings from /Users/lihongxuan/Desktop/AIPI/Courses/AIPI540/AIPI-540/wandb/settings
2024-02-04 18:10:18,492 INFO    MainThread:26015 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-04 18:10:18,492 INFO    MainThread:26015 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-04 18:10:18,492 INFO    MainThread:26015 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-04 18:10:18,492 INFO    MainThread:26015 [wandb_init.py:_log_setup():526] Logging user logs to /Users/lihongxuan/Desktop/AIPI/Courses/AIPI540/AIPI-540/wandb/run-20240204_181018-qg1xudow/logs/debug.log
2024-02-04 18:10:18,492 INFO    MainThread:26015 [wandb_init.py:_log_setup():527] Logging internal logs to /Users/lihongxuan/Desktop/AIPI/Courses/AIPI540/AIPI-540/wandb/run-20240204_181018-qg1xudow/logs/debug-internal.log
2024-02-04 18:10:18,492 INFO    MainThread:26015 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f87a829a040>
2024-02-04 18:10:18,493 INFO    MainThread:26015 [wandb_init.py:init():566] calling init triggers
2024-02-04 18:10:18,493 INFO    MainThread:26015 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-02-04 18:10:18,493 INFO    MainThread:26015 [wandb_init.py:init():591] re-initializing run, found existing run on stack: 8gkiu4hf
2024-02-04 18:10:18,494 INFO    MainThread:26015 [wandb_run.py:_finish():1962] finishing run aipi549/aipi540/8gkiu4hf
2024-02-04 18:10:18,495 INFO    MainThread:26015 [jupyter.py:save_history():445] not saving jupyter history
2024-02-04 18:10:18,495 INFO    MainThread:26015 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-04 18:10:18,495 INFO    MainThread:26015 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
